src/Adapter/EntityTranslation/TabTranslator.php

```diff
@@ -69,7 +69,7 @@ private function buildIndex(): array
     {
         $tableName = $this->dbPrefix . 'tab';
 
-        $sql = "SELECT id_tab, wording, wording_domain FROM $tableName";
+        $sql = "SELECT id_tab, wording, wording_domain FROM $tableName WHERE wording > '' and wording_domain > ''";
```

Comment:
Just curious: why `>` instead of `!=` ?

Reply:
`> ''` will not integrate `NULL` values in the result, it aims to check only wording with at least 1 character.
`!= ''` will this time integrate `NULL` values ^^

Reply:
Both are equivalent: http://sqlfiddle.com/#!9/e4805a/1

Reply:
Oh, funny, I retrieve null values from `!= ''` when I tested it :thinking: 
My bad :sweat_smile: